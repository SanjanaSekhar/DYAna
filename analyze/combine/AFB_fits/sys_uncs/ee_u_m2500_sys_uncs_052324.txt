Systematic uncertainties for bin 0
Electron Identification/Isolation & 0.58228 & 55.40  \\ 
LQ LO reweighting & 0.48765 & 38.86  \\ 
$\alpha_s$ + Renormalization/Factorization Scales & 0.14101 & 3.25  \\ 
Electron Momentum Scale & 0.08251 & 1.11  \\ 
Electron Trigger & 0.07431 & 0.90  \\ 
DY Cross Section & 0.05396 & 0.48  \\ 
Total Uncertainty & 0.78  \\ 
