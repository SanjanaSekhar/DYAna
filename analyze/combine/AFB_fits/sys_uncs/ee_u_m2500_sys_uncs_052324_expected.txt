Systematic uncertainties for bin 0
Electron Identification/Isolation & 0.58409 & 26.11  \\ 
LQ LO reweighting & 0.58139 & 25.87  \\ 
DY Cross Section & 0.52340 & 20.97  \\ 
Electron Momentum Scale & 0.52252 & 20.90  \\ 
$\ttbar$ Cross Section & 0.20159 & 3.11  \\ 
$\alpha_s$ + Renormalization/Factorization Scales & 0.19910 & 3.03  \\ 
Total Uncertainty & 1.14  \\ 
